src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -161,12 +174,15 @@ private function processNodeWithType(NodeInterface $node, NodeType $nodeTypeDefi
             }
 
             if (!$node->hasProperty($propertyDef->getName())) {
-                if ($propertyDef->isMandatory() && !$propertyDef->isAutoCreated()) {
+                if ($propertyDef->isMandatory()
+                    && !$propertyDef->isAutoCreated()
+                    && $nodeTypeDefinition->getName() !== VersionHandler::MIX_VERSIONABLE
```

Comment:
this means all mandatory non autocreated properties of the versionable mixin should not trigger an exception, right? did we not just add those on line 140? what will be added later? if this is still needed, lets put a comment line above the `&& $nodeTypeDefinition->getName()...` line explaining why we skip here and when they will be created.